css/src/components/form/checkbox.scss

```diff
@@ -116,4 +142,26 @@ $checkbox-spacing: $spacer-3 !default;
 			border-color: $checkbox-check-color;
 		}
 	}
+
+	/* stylelint-disable no-descending-specificity */
+
+	&.checkbox-danger input[type='checkbox']:not(:checked) {
```

Comment:
`.checkbox-danger` and `.checkbox-success` naming patterns don't match standard component naming conventions and style applications. If they did then the whole checkbox (including checked state) would be green or red in these cases. We also have a convention in atlas where states (focus, loading, etc) use `is-` classes. Should these modifiers be `is-invalid` and `is-valid` instead?

Reply:
I was debating about this as well, but decided to follow the same naming convention as other form components. But you're totally right - it's not fully meeting the expectation of turning the whole checkbox into red/green when selected. So updated per your suggestion.